src/components/Buttons.js

```diff
@@ -1,44 +1,36 @@
 import React from "react";
```

Comment:
FYI, since React v17 this import can be omitted. It is not wrong to have it, it is just not necessary.